core/state_transition.go

```diff
@@ -650,6 +665,9 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 				}
 				st.state.AddBalance(params.OptimismL1FeeRecipient, amtU256, tracing.BalanceIncreaseRewardTransactionFee)
 			}
+			if worstOperatorCost != nil {
```

Comment:
i still do not understand this - shouldnt the refund be subtracted?

Reply:
Not here; This is a reward dispersal. Adding balance to the fee _recipient_ is the right thing to do, but I'm not sure that the right amount of balance is being added here: https://github.com/ethereum-optimism/op-geth/pull/388#discussion_r1943305296